modules/packages/vendored/apt_get.mustache

```diff
@@ -32,7 +31,8 @@ apt_get_options = ["-o", "Dpkg::Options::=--force-confold",
                    "-o", "Dpkg::Options::=--force-confdef",
                    "-y"]
 
-if LooseVersion(apt_version) < LooseVersion("1.1"):
+# assume apt-get -v returns faily simple version with integers separated by dots
+if [int(x) for x in apt_version.split(".")] < [1, 1]:
```

Comment:
I'm not a pythonista, but this looks ok to me.
apt_get -v si processed a few lines up, getting the first line aplitting on spaces and taking the second (first if counting from 0) field. Then your if checks each part of the dotted version.
not clear to me though how the last dotted version is compared or what it's compared against.

```
❯ apt-get -v
apt 2.4.5 (amd64)
Supported modules:
*Ver: Standard .deb
 Pkg:  Debian APT solver interface (Priority -1000)
 Pkg:  Debian APT planner interface (Priority -1000)
*Pkg:  Debian dpkg interface (Priority 30)
 S.L: 'deb' Debian binary tree
 S.L: 'deb-src' Debian source tree
 Idx: EDSP scenario file
 Idx: EIPP scenario file
 Idx: Debian Source Index
 Idx: Debian Package Index
 Idx: Debian Translation Index
 Idx: Debian dpkg status file
 Idx: Debian deb file
 Idx: Debian dsc file
 Idx: Debian control file
```

Reply:
lists in python are compared left to right.